fake-cucumber/javascript/src/FakeStepDefinitions.ts

```diff
@@ -0,0 +1,103 @@
+import { messages } from 'cucumber-messages'
+import {
+  ParameterTypeRegistry
+} from 'cucumber-expressions'
+import uuidv4 from 'uuid/v4'
+
+import Argument from 'cucumber-expressions/dist/src/Argument'
+import CucumberExpression from 'cucumber-expressions/dist/src/CucumberExpression'
+
+class FakeStepDefinition {
+  public id: string
+  public expression: CucumberExpression
+
+  constructor(
+    public status: string,
+    public pattern: string = null
+  ) {
+    this.id = uuidv4()
+    if (pattern === null) {
+      this.pattern = `{}${status}{}`
+    }
+    this.expression = new CucumberExpression(this.pattern, new ParameterTypeRegistry)
```

Comment:
Use parenthesis when calling constructors:

```typescript
new ParameterTypeRegistry()
```

I also think it is cleaner to pass the whole `expression` object through the constructor, which should declare the type as `Expression` (imported from `cucumber-expressions`). That way we could also pass in a `RegularExpression` if we want, and all the expressions can share the same `ParameterTypeRegistry` instance (since it's created outside)